proposals/0313-drop-unchained-merkle-shreds.md

```diff
@@ -0,0 +1,76 @@
+---
+simd: '0313'
+title: Drop unchained merkle shreds
+authors:
+  - Ashwin Sekar
+category: Standard
+type: Core
+status: Review
+created: 2025-06-30
+feature:
+  - 5KLGJSASDVxKPjLCDWNtnABLpZjsQSrYZ8HKwcEdAMC8
+  - https://github.com/anza-xyz/feature-gate-tracker/issues/80
+---
+
+## Summary
+
+As the new shred format is being sent by all clients, this feature deprecates
+the old shred format.
+
+## Motivation
+
+It is inconvinient to support both formats, and upcoming consensus changes (alpenglow)
+expect all shreds to be of the chained variety.
+
+## New Terminology
+
+N/A
+
+## Detailed Design
+
+The chained merkle shred variant, adds an additional field to Data and Coding shreds.
+It adds the Merkle root of the previous erasure batch after the data buffer or
+erasure coded shard respectively, before the merkle proof:
+
+```
+// Layout: {common, data} headers | data buffer
+//     | [Merkle root of the previous erasure batch if chained]
+//     | Merkle proof
+//     | [Retransmitter's signature if resigned]
+// The slice past signature till the end of the data buffer is erasure coded.
+// The slice past signature and before the merkle proof is hashed to generate
+// the Merkle tree. The root of the Merkle tree is signed.
+
+// Layout: {common, data} headers | data buffer
+//     | [Merkle root of the previous erasure batch if chained]
```

Comment:
In context of this SIMD all shreds are chained, right?

Reply:
All shreds sent are now chained, but they used to not be chained. As specified below when the feature flag is active, any shred without the chained flag set will be dropped

Reply:
this makes sense, thank you!